### DISCUSSION TITLE
支付宝已付款续期1年，使用期限没有延长1年，麻烦解决一下，谢谢。

### DISCUSSION THREAD

**Discussion by aladdindune** [getlantern/lantern-forum-cn#919] [Q&A]
![1727837607809](https://github.com/user-attachments/assets/bbfeb8fb-d35b-4c23-aa5d-3e280874d5e3)
![1727837607818](https://github.com/user-attachments/assets/8f539c69-eae9-451e-b403-9eb0d3dfdd7d)

**Comment by aladdindune** [comment]
我的邮箱：aladdindune@hotmail.com

**Comment by MaxMKMax** [comment]
@aladdindune 您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.

**Comment by aladdindune** [comment]
您好，我提交的问题还没有解决。麻烦您催促一下，谢谢！

**Comment by aladdindune** [comment]
您好，我提交的问题什么时候能解决？时间也太长了吧！

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/f3e59f4a-3931-4559-be40-c6aacce4c0e8)
@aladdindune 您好. 问题几天前应该已经解决了. 您依然有同样的问题吗?

**Comment by aladdindune** [comment]
您好，确实没解决。还是没延长日期。麻烦您啦！谢谢！

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
